msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.NativeReference.Stubs.targets

```diff
@@ -0,0 +1,36 @@
+<!--
+***********************************************************************************************
+Xamarin.iOS.NativeReference.Stubs.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+  created a backup copy.  Incorrect changes to this file will make it
+  impossible to load or build your projects from the command-line or the IDE.
+
+This file imports the version- and platform-specific targets for the project importing
+this file. This file also defines targets to produce an error if the specified targets
+file does not exist, but the project is built anyway (command-line or IDE build).
+
+Copyright (C) 2013-2016 Xamarin Inc. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<!-- These "stub" targets are declared to prevent other (Windows specific) external targets from firing.
+	     They are required as we share use of @(NativeReference) with ClickOnce, and if we don't stub then it can do the wrong thing.
+	     Example: https://github.com/xamarin/xamarin-macios/issues/3876
+	     We copy the Condition/DependsOnTargets so our stub don't break on user targets that hook into them for ordering
+	-->
+	<Target Name="GenerateManifests"
+	      Condition="'$(GenerateClickOnceManifests)'=='true' or '@(NativeReference)'!='' or '@(ResolvedIsolatedComModules)'!='' or '$(GenerateAppxManifest)' == 'true'"
+	      DependsOnTargets="$(GenerateManifestsDependsOn)"/>
+
+	<Target Name="GenerateApplicationManifest" 
+		DependsOnTargets="_DeploymentComputeNativeManifestInfo;_DeploymentComputeClickOnceManifestInfo;ResolveComReferences;ResolveNativeReferences;_GenerateResolvedDeploymentManifestEntryPoint"
```

Comment:
On second though on this particular case, I don't think we want any of those depends on. None of them actually do anything useful in an iOS project and would just run unnecessarily. In particular I think I saw `ResolveNativeReferences` trying to load/read something from the files, which didn't apply at all and I think caused some warnings. `ResolveProjectReferences` below is different, since it's something that's also done for iOS projects, and the ordering might be altered if we don't include it, but in this case, I think we can remove them all, actually.